packages/financial-templates-lib/src/price-feed/InsuredBridgePriceFeed.ts

```diff
@@ -0,0 +1,194 @@
+import assert from "assert";
+import { PriceFeedInterface } from "./PriceFeedInterface";
+import Web3 from "web3";
+import { parseAncillaryData } from "@uma/common";
+import { getAbi } from "@uma/contracts-node";
+import { BN } from "../types";
+import type { Logger } from "winston";
+import { InsuredBridgeL1Client } from "../clients/InsuredBridgeL1Client";
+import { InsuredBridgeL2Client } from "../clients/InsuredBridgeL2Client";
+
+const { toBN, isAddress } = Web3.utils;
+
+enum relayState {
+  Pending,
+  SpedUp,
+  Disputed,
+  Finalized,
+}
+
+enum isRelayValid {
+  Yes,
+  No,
+}
+
+interface Params {
+  logger: Logger;
+  web3: Web3;
+  bridgeAdminAddress: string;
+}
+
+interface Relay {
```

Comment:
nit: a lot of these interfaces are exported from the clients. consider importing them from there rather than re-defining them here.

Reply:
see here: https://github.com/UMAprotocol/protocol/blob/a350bd9d1fc9a8c58b4a57f58fee62e7cfd75141/packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts#L6 as an example of this import

Reply:
Got it, done